src/demo/index.tsx

```diff
@@ -1,6 +1,8 @@
 import * as React from 'react'
 import { render } from 'react-dom'
 import { ReactTinyLink } from '../../lib/index'
+import YoutubeLinkIssue from './YoutubeLinkIssue'
```

Comment:
Hi @hitesh399 
Great work,Thanks for the contribution, 
One request from me can you help me to remove this as i didnt see the use of the 'YoutubeLinkIssue' component and source.